python/ray/serve/tests/test_api.py

```diff
@@ -774,7 +774,11 @@ def batcher(flask_requests):
     ray.get([block_until_http_ready.remote(url) for _ in range(10)])
 
     def verify_metrics(do_assert=False):
-        resp = requests.get("http://127.0.0.1:9999").text
+        try:
+            resp = requests.get("http://127.0.0.1:9999").text
+        # Requests will fail if we are crashing the controller
```

Comment:
nice. but test_api won't be run in bazel with crash enabled?

Reply:
as, the bazel rule 
```
py_test(
    name = "test_controller_crashes",
    size = "large",
```
will only run the file named `test_controller_crashes.py`, according to bazel's rule.

Reply:
No it will, `test_controller_crashes` calls pytest on all the files that it is globbed with.

Reply:
ahhhh i see. test_controller_crashes just do globbing and uses the same pattern we used to do in test_serve. 🤦 sorry about the confusion

Reply:
No worries!